task_set.json

```diff
@@ -76,11 +144,5 @@
             "sentiment-scoring",
             "other"
         ]
-    },
-    "other": {
```

Comment:
I wouldn't remove "other" just yet, it's hard to guarantee we currently have exhausted all the possible tasks :)

Reply:
"other"  is in between. This is a sorted JSON.  :P